### ISSUE TITLE
`Cmd+W` with no open tabs doesn't close the window

### ISSUE THREAD

**Issue by adamsoutar** [zed-industries/feedback#218]
**Is your feature request related to a problem? Please describe.**

Just a tiny nit (feel free to say if these tiny bugs aren't helpful), but when the window is empty, I would have expected `Cmd+W` to act like `Cmd+Shift+W`, but it currently does nothing.

(See: behaviour in Atom, VS Code)

**Comment by fuzenco** [comment]
Yes, it’s one of those little things that would make Zed better for me. I frequently work with just single files at a time. Having that last window automatically close would be awesome.
